_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,13 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Improved keyboard navigation for screen readers.
```

Comment:
Contemplating if we should add more detail here. It might be helpful if we highlight some of the specific changes like enabling table screen reader controls.

Reply:
Took a stab with [d143905](https://github.com/uswds/uswds-site/pull/2218/commits/d143905cfbd22b971d24c84c0644dc01c057244c). Let me know what you think!